fares-data-build-tool/fdbt-site
# Changed logic to only work for ticketer format CSVs

no longer removing column A data and updated format of template to match

# Description

-   To allow users to upload CSV's in the ticketer format, and change our example CSV format to match.

# Testing instructions

-   Upload a new format CSV and observe it working as normal. Upload an old format CSV and see it not work.

# Type of change


# Checklist:
